packages/material/README.md

```diff
@@ -1,24 +1,29 @@
-# JSONForms - More Forms. Less Code
+# JSON Forms - More Forms. Less Code
+
 ### Complex Forms in the blink of an eye
 
-JSONForms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+JSON Forms eliminates the tedious task of writing fully-featured forms by hand by leveraging the capabilities of JSON, JSON Schema and Javascript.
+
+## Material Renderers Package
+
+This is the JSON Forms Material Renderers Package. This package only contains renderers and must be combined with [JSON Forms React](https://www.npmjs.com/package/@jsonforms/react).
+
```

Comment:
Also mention and link the React seed and the React documentation on JSON forms website.